gkeys/gkeys/seedhandler.py

```diff
@@ -66,8 +66,11 @@ def build_gkeydict(args):
                     keyinfo[attr] = decoder(value)
                 else:
                     keyinfo[attr] = value
-            except AttributeError:
-                pass
+            except (AttributeError):
+                if attr in GKEY._fields:
+                    keyinfo[attr] = None
+                else:
```

Comment:
If there isn't any action the `else`statement can be omitted and you can keep only the `if` statement.